### DISCUSSION TITLE
Not able to deploy application through Argo-CD in Kubernetes cluster

### DISCUSSION THREAD

**Discussion by Nayana-Nagaraj** [argoproj/argo-cd#10768] [General]
Hello Team,

I am trying to deploy through argocd in kubernetes cluster. But unfortunately i am getting the below error: 
rpc error: code = DeadlineExceeded desc = context deadline exceeded 

![image](https://user-images.githubusercontent.com/96725710/193581972-6bc9d7f4-9e56-45b5-9ee5-109679f3c443.png)

And also i am getting the name error when trying to connect with git repository.
![image](https://user-images.githubusercontent.com/96725710/193582174-27de0c10-d0a2-43ac-9622-e29667916b51.png)

Can you please help me to resolve this

**Comment by jgwest** [comment]
You may want to verify that all the components of your Argo CD installation are healthy, for example, the repo server pod of Argo CD is up and running, and not crash looping. Likewise, the logs for the Argo CD pods must contain useful additional information. Otherwise, the next thing I would suggest is verify that the Git server you are attempting to connect to is accessible from the K8s cluster, and the Git server is returning a response in a timely fashion.
